### PR TITLE
[release-1.19] Fix issue with dual-stack iptables6 rules when using istio-cni plugin…

### PR DESCRIPTION
**Pull request by jacob-delgado**
Fixes https://github.com/istio/istio/issues/47056

cherry pick of https://github.com/istio/istio/pull/46630

### CODE REVIEW

**Review by hanxiaop — APPROVED** [review]

**Comment by hanxiaop** [comment]
@jacob-delgado I thought `TestReachability` was a test flake but it looks like it's not. Is there anything that hasn't been cherry-picked?

File: tools/istio-iptables/pkg/cmd/root.go [hunk 1]
```diff
@@ -408,6 +410,11 @@ func bindFlags(cmd *cobra.Command, args []string) {
 		handleError(err)
 	}
 	viper.SetDefault(constants.HostNSEnterExec, false)
+
+	if err := viper.BindPFlag(constants.DualStack, cmd.Flags().Lookup(constants.DualStack)); err != nil {
+		handleError(err)
+	}
+	viper.SetDefault(constants.DualStack, false)
```
Comment (adams-shaun) [inline tools/istio-iptables/pkg/cmd/root.go:417]: default to `DualStack`

**Comment by adams-shaun** [comment]
@hanxiaop @jacob-delgado - opened new PR here https://github.com/istio/istio/pull/47108

**Pull request closed without merging**